packages/ui/src/working-groups/model/getNextPayout.ts

```diff
@@ -12,7 +12,7 @@ export function getNextPayout(workers: Pick<Worker, 'group'>[], blockNumber: BN,
   const nextPayoutPerGroup = userGroups.map((name) => api?.consts[name].rewardPeriod.toBn()).map(blocksUntilNext)
 
   if (nextPayoutPerGroup.length) {
-    return nextPayoutPerGroup.reduce((closest, time) => (closest = BN.min(closest ?? BN_ZERO, time ?? BN_MILLION)))
+    return nextPayoutPerGroup.reduce((closest, time) => (closest = BN.min(closest ?? BN_ZERO, time ?? BN_MAX_INTEGER)))
   }
   return new BN(-1)
 }
```

Comment:
Now that you edited this file. Here's what I'm proposing:
```ts
const nextPayoutPerGroup = userGroups.flatMap((name) => {
  const interval = api?.consts[name].rewardPeriod.toNumber()
  return interval ? [interval - (blockNumber % interval)] : []
}

if (nextPayoutPerGroup.length === 0) {
  return
}

return Math.min(...nextPayoutPerGroup)
```
Of course the returned type is now different so a few adjustment are necessary but at least the values should always make sense now.

Reply:
I changed it the way you do.

Reply:
@chrlschwb I really think the `getNextPayout` function should be rewritten, this code is confusing. However it's not really related to this PR which appear to fix what it's supposed to fix.

So please just revert the changes on this file, because although they are both wrong at least `BN_MILLION` blocks makes more sense than `BN_MAX_INTEGER` blocks. I'll fix this function in another PR.